src/IIS/Settings/FtpsiteSettings.cs

```diff
@@ -6,8 +6,7 @@ public class FtpsiteSettings : SiteSettings
             public FtpsiteSettings()
                 : base()
             {
-                this.BindingProtocol               = BindingProtocol.Ftp;
-                this.Port                          = 21;
+                ChangeBindingTo().Ftp();
```

Comment:
This could be as simple as this.Binding = IISBindings.Ftp() if you achanged the interfaces to expose properties and made IISBindings static